src/mixins/Projector.ts

```diff
@@ -215,14 +218,22 @@ export function ProjectorMixin<P, T extends Constructor<WidgetBase<P>>>(base: T)
 		}
 
 		public setChildren(children: DNode[]): void {
+			this._projectorChildren = [ ...children ];
```

Comment:
We are taking a copy at the point in time the user has explicitly called `setProperties` or `setChildren`, because the properties and children that are passed can be decorated in lifecycle phases such as `beforeRender` - on a re-render for all other widgets in the widget tree a "fresh" properties object is used via the usage of `w()` but for the projector that is not the case and therefore the next render the possibly modified `properties` are used again (which means it can cause unexpected behaviour). So with these "copied" local value we re-set the `properties`/`children` in `WidgetBase` by calling the setters.

We could actually directly call the supers `__setProperties__` and `__setChildren__` from the `__render__` method as we only need the to take the copy when the user has actually called the methods intended for public consumption `setProperties` and `setChildren` - I can change that.